src/components/i18nTextarea.vue

```diff
@@ -130,27 +115,28 @@ export default {
 		},
 	},
 	methods: {
+		userRequestedNewLanguage: function(lang) {
+			this.addLanguage(lang)
+			this.focusOnLastTab()
+		},
 		changeText(key, value) {
 			const el = this.$refs['textarea-tab-' + key][0].$el
 			autosize(el)
 
 			this.$set(this.state, key, value)
 			this.updateValue()
 		},
-		addTab(lang) {
+		addLanguage(lang) {
 			if (lang in this.state) {
-				this.focusOnTabWithLanguage(lang)
 				return
 			}
 			this.$set(this.state, lang, '')
-			this.updateValue()
-			this.focusOnLastTab()
 		},
 		deleteLang(lang) {
 			this.$delete(this.state, lang)
-			this.updateValue()
 		},
 		updateValue() {
+			console.log("update value called i18ntextarea")
```

Comment:
console.log

Reply:
true.. used those to follow the updateValue triggers :D

Reply:
removed